src/templates/subpageTemplate.js

```diff
@@ -1,18 +1,25 @@
 import React from "react"
 import { graphql } from "gatsby"
 
-import f from '../scss/modules/foundation.module.scss';
 import s from '../scss/modules/subpage.module.scss';
 
+import Helmet from '../components/Helmet';
 import Subpage from '../components/Subpage';
 
 export default function SubpageTemplate({
   data, // this prop will be injected by the GraphQL query below.
 }) {
-  const { markdownRemark } = data // data.markdownRemark holds our post data
+  const { markdownRemark, site } = data // data.markdownRemark holds our post data
   const { frontmatter, html } = markdownRemark
   return (
     <Subpage>
+      <Helmet
+        meta={[
+          { name: 'description', content: 'Dynamic, Next Generation Legal Agreements' },
+        ]}
+        path={frontmatter.path.replace(/^\//, '')}
+        title={`${site.siteMetadata.title} — A free legal repository — ${frontmatter.title}`}
```

Comment:
Could we switch this to have the page title 1st? 

`title={`${frontmatter.title} - ${site.siteMetadata.title} - A free legal repository`}`

Or, `title={`${frontmatter.title} | ${site.siteMetadata.title} - A free legal repository`}`

Not sure if you or @atgrif has a preference on bar vs. hyphen for page titles.